storagenode/storagenodedb/storagenodedbtest/run_test.go

```diff
@@ -30,6 +31,83 @@ func TestDatabase(t *testing.T) {
 	})
 }
 
+func TestBandwidthRollup(t *testing.T) {
```

Comment:
This test is in the wrong place, the tests about behavior should be near the interface not the implementation.